es2015.md

```diff
@@ -397,7 +412,7 @@ An exported variable binding, e.g., `{foo}` in `export {foo}` or `{bar as foo}`
 ```js
 interface ExportDefaultDeclaration <: ModuleDeclaration {
     type: "ExportDefaultDeclaration";
-    declaration: Declaration | Expression;
+    declaration: OptIdFunctionDeclaration | OptIdClassDeclaration | VariableDeclaration | Expression;
```

Comment:
I think `VariableDeclaration` is disallowed here: https://www.ecma-international.org/ecma-262/8.0/#prod-ExportDeclaration

Reply:
Agreed.